pkg/controller/backup_operations_controller.go

```diff
@@ -82,9 +82,14 @@ func NewBackupOperationsReconciler(
 
 func (c *backupOperationsReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	s := kube.NewPeriodicalEnqueueSource(c.logger, mgr.GetClient(), &velerov1api.BackupList{}, c.frequency, kube.PeriodicalEnqueueSourceOption{})
+	gp := kube.NewGenericEventPredicate(func(object client.Object) bool {
+		restore := object.(*velerov1api.Restore)
```

Comment:
It should be `backup := object.(*velerov1api.Backup)` because we want to filter backup in backupOperationsReconciler

Reply:
@Lyndon-Li oops. Sorry, copy-and-paste error from the similar fix on the restore side. I just noticed the e2e test failures which are probably related to this.

Reply:
fixed